examples/basic_operations/get_responsive_search_ads.rb

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env ruby
+# Encoding: utf-8
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+# This example retrieves expanded text ads.
```

Comment:
Do we really need a get() example? Won't it suffice to state in the guide that you can use GoogleAdsService.Search, with the following query?

Reply:
We have get examples for other specific ad types like ETA and Gmail ad, I don't see why we wouldn't have a get example for this. I think adding the GAQL to the guide *in addition* to this example works.